[libplist] Unable to build

```
Making all in src
make[2]: Entering directory '/usr/local/tmp/crew/2.1.0.tar.gz.dir/libplist-2.1.0/src'
  CC       base64.lo
  CC       bytearray.lo
  CC       hashtable.lo
  CC       ptrarray.lo
  CC       time64.lo
  CC       xplist.lo
  CC       bplist.lo
  CC       plist.lo
  CXX      Node.lo
In file included from Node.cpp:22:
../include/plist/Node.h:26:10: fatal error: 'cstddef' file not found
#include <cstddef>
         ^~~~~~~~~
1 error generated.
make[2]: *** [Makefile:600: Node.lo] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/usr/local/tmp/crew/2.1.0.tar.gz.dir/libplist-2.1.0/src'
make[1]: *** [Makefile:438: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/2.1.0.tar.gz.dir/libplist-2.1.0'
make: *** [Makefile:368: all] Error 2
libplist failed to build: `make V=0 -j2` exited with 2
```

--- a/packages/lftp.rb
+++ b/packages/lftp.rb
@@ -8,17 +8,20 @@ class Lftp < Package
   source_sha256 '5969fcaefd102955dd882f3bcd8962198bc537224749ed92f206f415207a024b'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/lftp-4.9.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/lftp-4.9.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/lftp-4.9.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/lftp-4.9.1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'a803ec42009f43031b2d77d8ffe88933877d5d921fcb4bb093dd0e366ca3cfb1',
+     armv7l: 'a803ec42009f43031b2d77d8ffe88933877d5d921fcb4bb093dd0e366ca3cfb1',
+       i686: '2752fe7e4035e1847979379fcb1723a48bc9f06d6a37d3eda92c67fbe803ebdd',
+     x86_64: '6001988f90424bb072329bf8ff8f84410c17e65dbd06a90d43f27f323453b731',
   })
 
-
   def self.build
-    system "./configure \
-      --prefix=#{CREW_PREFIX} \
-      --libdir=#{CREW_LIB_PREFIX} \
-      --disable-nls \
-      --with-linux-crypto"
+    system "./configure #{CREW_OPTIONS} --disable-nls --with-linux-crypto"
     system 'make'
   end
 

--- a/packages/libplist.rb
+++ b/packages/libplist.rb
@@ -8,26 +8,30 @@ class Libplist < Package
   source_sha256 '4b33f9af3f9208d54a3c3e1a8c149932513f451c98d1dd696fe42c06e30b7f03'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libplist-2.1.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libplist-2.1.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libplist-2.1.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libplist-2.1.0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'adca559671e694422de5f311b9c968d4e7f5fc45aae8c1c9140e825741475d26',
+     armv7l: 'adca559671e694422de5f311b9c968d4e7f5fc45aae8c1c9140e825741475d26',
+       i686: '11f60dc59c427f286b717fcd63ada445effc478c5895f51130128a6b1ff5418e',
+     x86_64: 'd6580afa2ec93f6a41f95e31cb86dd5dd7828f09a5091323f2fe6a4ee90a4e20',
   })
 
   depends_on 'glib'
 
   def self.build
-    system './autogen.sh CC=gcc CXX=g++'
-    system './configure',
-      "--prefix=#{CREW_PREFIX}",
-      "--libdir=#{CREW_LIB_PREFIX}",
-      '--disable-dependency-tracking'
+    system "PYTHON=#{CREW_PREFIX}/bin/python3 ./autogen.sh CC=gcc CXX=g++ #{CREW_OPTIONS}"
     system 'make'
-  end
-
-  def self.install
-    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
   end
 
   def self.check
     system 'make', 'check'
+  end
+
+  def self.install
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
   end
 end

